app/renderer/components/navigation/urlBar.js

```diff
@@ -428,6 +398,16 @@ class UrlBar extends React.Component {
     contextMenus.onUrlBarContextMenu(e)
   }
 
+  onStop () {
+    ipc.emit(messages.SHORTCUT_ACTIVE_FRAME_STOP)
```

Comment:
I don't like duplicating this logic here. Can we create `windowActions.onStop` and have both methods call that instead?